backend/lib/endpoints/schema/posts.js

```diff
@@ -18,6 +18,7 @@ const getPostsSchema = {
     .prop("ignoreUserLocation", S.boolean().default(false))
     .prop("objective", S.string().enum(POST_OBJECTIVES))
     .prop("skip", S.integer())
+    .prop("postMode", S.string())
```

Comment:
here can use the enum POST_MODE defined above to validate input

Reply:
like mentioned above, using a enum type can help to validate the input here in the schema